docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,130 @@
+# Faker connector
+
+The Faker connector generates realistically looking random data matching a
+specific schema.
+
+## Configuration
+
+To configure the Faker connector, create a catalog properties file
+`etc/catalog/example.properties` with the following contents:
+
+```text
+connector.name=faker
+```
```

Comment:
And there is no other configuration whatsoever?

Reply:
There are only two, mentioned in the following sections. I'll add them in this example, since changing their defaults should be relatively common.